PiwikTracker.php

```diff
@@ -1288,6 +1288,25 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param $proxy
+     * @param int $proxyPort
+     */
+    public function setProxy($proxy, $proxyPort = 80){
+        $this->proxy = $proxy;
+        $this->proxyPort = $proxyPort;
+    }
+
+    public function getProxy(){
```

Comment:
I wonder if we need this method public or if we can have it private? If it's public can you maybe add a little description and phpdoc?